google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -399,17 +406,47 @@ public void deleteSession(String sessionName, @Nullable Map<Option, ?> options)
   }
 
   @Override
-  public ServerStream<PartialResultSet> read(
+  public StreamingCall read(
```

Comment:
It's not reverting back. The previous PR migrates the traffic to gapic but disabled resuming and preferedChunks while this PR fixed them. If it would help you could compare this PR with master.